domain/src/main/java/life/qbic/usermanagement/User.java

```diff
@@ -6,11 +6,12 @@
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import jdk.jshell.spi.ExecutionControl.UserException;
 import life.qbic.usermanagement.policies.*;
 
 /**
  * <b>User class</b>
- *
+ * <p>
  * User aggregate in the context of user management.
  *
```

Comment:
```suggestion
 * </p>
```

Reply:
That's how its done when using the auto refactor from IntelliJ. We should make use of it and not fight against the tools

Reply:
but is it then properly rendered?

Reply:
I think we need the closing tag here

Reply:
not needed, you can check out and see for yourself :)